packages/uix-host/src/port.ts

```diff
@@ -435,10 +435,7 @@ export class Port<GuestApi = unknown>
       try {
         methodCallee = this.getHostMethodCallee(address, privateMethods);
       } catch (e) {
-        this.logger.warn(
-          `Private method ${formatHostMethodAddress(address)} not found!`,
-          address
-        );
+        // ignore error, privateMethod gets called and works
```

Comment:
@nitinja , If we decide to remove this warning message, then this comment needs to be changed. We cannot state that the private method has been called, as apparently it hasn't been found.

Reply:
This was an oversight, you are right. I have now amended this message to refelect it.